pkgs/applications/editors/neovim/default.nix

```diff
@@ -108,6 +108,7 @@ let
                 $out/bin/nvim
     '' + optionalString withPython ''
       ln -s ${pythonEnv}/bin/python $out/bin/nvim-python
+      ln -s ${pythonEnv}/bin/python $out/bin/python2
```

Comment:
this shouldn't be renamed to python2 since it will collide with any other pythonX package. we dont want that. we can still wrap it.